Neutral models of quantitative trait evolution are important for establishing a
baseline against which to test for selection. \citet{Schraiber2015} recently
analyzed a neutral model of trait evolution that made few assumptions about the
number of loci potentially affecting the trait and the distribution of
mutational effects. However, they only derived results for constant-size,
panmictic populations. We extend their results to populations with arbitrary
distributions of coalescent times and therefore arbitrary demographic histories
and spatial structures. A key result of \citet{Schraiber2015} is the
characteristic function of the distribution of trait values in a sample. We work
instead with the moment generating function, but the two approaches are
interchangeable as long as the mgf of the mutational distribution exists (which
it will if the moments of the mutational distribution are all finite). Our main
result (equation \eqref{eq:sub}) shows that the characteristic function
from \citet{Schraiber2015} is a special outcome of a general procedure whereby
the mgf for a trait can be obtained by making a simple substitution into the mgf
for a distribution over genealogies. The mgfs for many demographies of interest
and for sample sizes above three are complex recursions that are impractical to
solve \citep{Lohse2011}. However, progress can still be made by deriving moments
of trait distributions in terms of moments of the genealogical and mutational
distributions.

This result extends previous work using coalescent theory to investigate neutral
models of quantitative traits \citep{Whitlock1999,Schraiber2015}. Ours is the
most general model yet analyzed. As a natural first step, we show that the
infinitesimal limit suggested by \citet{Fisher1918} leads to a model where
phenotypes are normally distributed as the number of loci becomes large and the
magnitude of effect sizes becomes small. In the limiting distribution, the
variance of the difference between two individuals is proportional to the
expected pairwise coalescent time between them, and the covariance between a
pair of differences is $\Cov[Y_a - Y_b,Y_c - Y_d] \propto \E[\mathcal{T}_{a,d}]
+ \E[\mathcal{T}_{b,c}] -
\E[\mathcal{T}_{a,c}] - \E[\mathcal{T}_{b,d}]$. The resulting covariance matrix
completely specifies the neutral distribution in the infinitesimal limit. This
is similar to classic models in evolutionary quantitative genetics for the
neutral divergence of trait values after population splits
\citep{Lande1976,Lynch1989} but holds regardless of the precise details of
population structure and history. \citet{Schraiber2015} derive essentially the
same distribution using a central limit theorem argument.

\citet{Mendes2018} point out several problems caused by incomplete
lineage sorting when a covariance matrix based on species split times is used in
phylogenetic comparative methods. The normal model arising in the infinitesimal
limit is not subject to these problems because it is explicitly based in the
coalescent. A matrix based on average pairwise coalescence times takes into
account the effects of all lineages at causal loci, even those that do not
follow the species topology. The covariances specified by
equation \eqref{eq:normcov} could also be used to generate within-species
contrasts in a similar manner to the method suggested
by \citet{Felsenstein2002}.

We derived the first four expected central moments of sparse traits and compared
them to those expected under normality. This showed how demography and genetics
separately influence the expected deviation from normality. For a fixed expected
sparsity, population growth produces greater deviations in the fourth central
moment while population bottlenecks produce lower deviations (Figure
\ref{fig:Qexp}). However, for realistic demographic scenarios,
we find that the effects attributable to demography are small (Figure
\ref{fig:afeucomp}). We only analyzed cases with exchangeable individuals,
but adding population structure would increase deviations from normality as
drifting trait means between subpopulations can yield multimodal distributions.

We next investigated two simple problems where a coalescent perspective on the
neutral distribution of a quantitative trait provides useful intuition. The
first is the question of the appropriate null distribution for $Q_{ST}$ at the
population level. We show how to easily simulate from the null distribution in
the infinitesimal limit, providing a much better approximation when
subpopulations are correlated than previous approaches \citep{Whitlock2009}
(Figure \ref{fig:qst_deme}). For the second we show that the shape of the
mutational distribution is largely confounded with the number of loci affecting
the trait, with only one compound parameter identifiable. This makes it unlikely
that mutational parameters could be inferred from trait values sampled from a
population.

Even though we have broadened the model space for neutral traits, many features
of real populations have not yet been incorporated. Linked loci are a particular
concern as there is substantial linkage disequilibrium between causal
SNPs \citep{Bulik-Sullivan2015}. \citet{Lohse2011} derived the form of the mgf
for genealogies at linked loci and future work will attempt to incorporate
recombination using equation \eqref{eq:sub}. In particular, it will be important
to show how linkage affects the distribution in the infinitesimal limit.
Diploidy, dominance, and epistasis have also been ignored thus far. The
qualitative effects described here should hold under diploidy, but having trait
values within individuals summed over loci from two copies of the genome will
decrease deviations from normality. Dominance will also tend to produce a normal
distribution as the effects are independent between loci, but work is needed to
examine how this interacts with the distribution of genealogies to affect the
trait distribution.

\citet{Barton2017} recently performed a deep mathematical investigation of a
more formal ``infinitesimal model''. They proved conditions under which the
trait values of offspring within a family are normally distributed with a
covariance matrix conditionally independent of the parental trait values given
the pedigree. Interestingly, they found normality still holds under pairwise
epistasis if it is not too extreme. It may therefore be possible to include
epistasis in the infinitesimal limit considered here and thus study how
epistasis affects neutral trait divergence. In a commentary on
\citet{Barton2017}, \citet{Turelli2017} noted that at least three
different sorts of infinitesimal models have been used. The infinitesimal limit
considered here fits all three: (1) the limit corresponds to Fisher's notion of
a large number of Mendelian factors, each making a small contribution
\citep{Fisher1918}; (2) the descendants within a family have a Gaussian
distribution of trait values, as shown rigorously by \citet{Barton2017}; and (3)
the distribution of trait values in a panmictic population is Gaussian.

Although GWAS of many traits have shown them to be controlled by large numbers
of loci \citep{Boyle2017}, this is not necessarily the case for every trait of
interest to biologists. It has been suggested, for instance, that gene
expression levels have a sparse genetic architecture \citep{Wheeler2016}. Since
there is much interest in testing whether natural selection has acted on gene
expression levels \citep{Whitehead2006,Gilad2006,Yang2017}, well-calibrated
goodness-of-fit tests will need to take into account the complications that
arise when trait distributions deviate from normality \citep{Khaitovich2005}.
Direct measurements of mutational distributions \citep{Gruber2012,Metzger2016}
could aid calibrations. Generally, more work is needed to develop neutrality
tests robust to the details of genetics and population history, and to
investigate whether anything about mutational processes can be learned using
statistical models that share information across multiple traits.

%%% Local Variables:
%%% TeX-master: "quant_gen_manu.tex"
%%% End:
